src/components/views/ScreeningsCard.vue

```diff
@@ -0,0 +1,121 @@
+<script>
+import { defineComponent } from 'vue';
+import MoviePoster from '@/components/common/Movie/MoviePoster.vue';
+import MovieLength from '@/components/common/Movie/MovieLength.vue';
+import AppButton from '@/components/common/App/AppButton.vue';
+import MovieTitle from '@/components/common/Movie/MovieTitle.vue';
+import AppTag from '@/components/common/App/AppTag.vue';
+import { getScreeningsByMovieAndDate } from '@/api/service/Screenings';
+
+export default defineComponent({
+  components: { MoviePoster, MovieLength, AppButton, MovieTitle, AppTag },
+  props : {
+    movieData: {
+      type: Object,
+      required: true,
+    },
+    daySelected: {
+      type: Date,
+      default: new Date().toISOString().slice(0, 10)
+    },
+    titleSelected: {
+      type: String,
+      default: 'All movies',
+    }
+  },
+  data() {
+    return {
+      screenings: [],
+      selectedMovieId: this.movieData.id,
+    }
+  },
+  watch: {
+    daySelected(newDay, previousDay) {
+      if (newDay !== previousDay) {
+        this.getScreenings()
+      }
+    },
+    titleSelected(newOption, previousOption) {
+      if (newOption !== previousOption) {
+        this.selectedMovieId = this.getMovieIdByTitle(this.movieData, newOption)
+        this.getScreenings()
+      }
+      if (newOption === 'All movies') {
+        this.selectedMovieId = this.movieData.id
+        this.getScreenings()
+      }
+    }
+  },
```

Comment:
It would be a much better idea to move this kind of logic to the parent component, so that you have a _filterable list_ rather than the _list of self-filterable items_. 

The screening card should only display the data, anything more is just too much. Make a computed property in the `ScreeningList` component and simply filter out the movies you don't want to be displayed.

Reply:
Ahhh... this was just a brilliant idea! ✨ The refactor made my brain boil, but IT WORKS! 🎉 Will commit it soon!

Reply:
Thanks a lot for explaining this mechanism! 🚀